website/docs/guides/qwik.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Qwik
+section: Frameworks
+---
+
+# Qwik
+
+---
+
+To use Fontsource with a [Qwik City](https://qwik.builder.io) project, it is suggested to use the `/routes/layout.tsx` file. 
+
+## Installation
```

Comment:
I think we should number the steps like the Remix or SvelteKit guide. I'd like all guides to be written in that format (even though some of the older ones don't).